etc/logging.yaml.journald.sample

```diff
@@ -0,0 +1,38 @@
+version: 1
```

Comment:
Add run_travis.sh to be executed while PRs are sent. Same applicable tp performance_monitoring as well.
@r0h4n  do we need some enabling of running travis for these projects?